### PR TITLE
25 predicted data page

### PR DESCRIPTION
**Pull request by DanBenesh**
Added PredictedDataPage directory in src/pages. Had to change Navbar references to match location.

### CODE REVIEW

File: frontend/yarn.lock
Comment (ralphgregorio) [inline frontend/yarn.lock:1]: Can you also revert your changes in this file as well

File: frontend/package.json
Comment (ralphgregorio) [inline frontend/package.json:1]: can you revert changes made into this file?

File: frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Button, Container, List } from "@mui/material";
+import styles from "./PredictedDataPage.module.css";
+
+function PredictedFiles(): React.ReactElement {
```
Comment (ralphgregorio) [inline frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx]: Function name should be the same as the file name.
```suggestion
function PredictedDataPage(): React.ReactElement {
```

File: frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+              </span>
+            </div>
+          </List>
+        </Container>
+      </div>
+    </div>
+  );
+}
+
+export default PredictedFiles;
```
Comment (ralphgregorio) [inline frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx]:
```suggestion
export default PredictedDataPage;
```

File: frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <Button variant="contained" className={styles.downloadBtn}>
+              Download File
+            </Button>
+          </div>
+          <List className={styles.list}>
+            <div className={styles.bannerinfo}>
+              <span>
+                If you want to upload different files, go back to the file
+                upload page. You can also click on individual steps to rerun a
+                different machine learning model if you want.
```
Comment (ralphgregorio) [inline frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx]: We should also add a test file for these.

For now we can just test if the text are actually rendered. 

HomePage.test.tsx is a good reference on how to make this. Or If you need can pair with you on making the test cases as well.

**Review by ralphgregorio — CHANGES_REQUESTED** [review]
needs a unit test and revert the changes on packages.json and yarn.lock.

Otherwise looks good!

File: frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx (hunk 3 above)
Comment (DanBenesh) [reply frontend/src/pages/PredictedDataPage/PredictedDataPage.tsx]: I have added the test page. I believe I had renamed everything to 'PredictedDataPage'. I'll check again

**Comment by DanBenesh** [comment]
Playwright tests need to be updated. Rendering page "PredictedFiles" needs to be changed to "PredictedDataPage"

File: package.json
Comment (ralphgregorio) [inline package.json:1]: i think this was added on accident again? can we delete this too?

**Review by ralphgregorio — APPROVED** [review]
thanks for the changes. looks good! just 1 minor comment

**Pull request merged**